11ty/definitions/colorblind.md

```diff
@@ -0,0 +1,44 @@
+---
+title: colorblind (race)
+slug: colorblind-racism
+flag:
+  text: 'Medical appropriation'
+  level: avoid
+defined: true
+excerpt: Colorblind, someone with decreased ability to see color or differences in color; colloquially, belief that racism is no longer a problem and that we all have equal opportunities.
+speech: adjective
+alt_words:
+  - ignorant
+  - insensitive
+  - post-racial
+  - race neutral
+reading:
+  - text: 'What you erase when you say “I don’t see color”'
+    href: https://medium.com/@tinu/what-you-erase-when-you-say-i-dont-see-color-73360346afa7
+  - text: '7 Reasons Why "Colorblindness" Contributes to Racism Instead of Solves It'
+    href: https://everydayfeminism.com/2015/02/colorblindness-adds-to-racism/
+---
+
+colorblind (or color-blind), a condition where someone can only see a limited range of colors or is unable to clearly distinguish different colors. Color blindness can be congenital (from birth) or acquired (due to chronic illness, disease, injury, or old age); colloquially, belief that racism is no longer a problem and that we all have equal opportunities.
```

Comment:
```suggestion
a condition where someone can only see a limited range of colors or is unable to clearly distinguish different colors; condition can be congenital (from birth) or acquired (due to chronic illness, disease, injury, old age, etc); colloquially, belief that "one does not see race," and, that racism was/no longer is a systemic problem; also post-racial.

(Also spelled colour-blind.)
```

Reply:
I made this change but didn't use the UK spelling of color (colour).